src/vs/workbench/contrib/files/browser/files.contribution.ts

```diff
@@ -398,6 +398,19 @@ configurationRegistry.registerConfiguration({
 			],
 			'description': nls.localize('sortOrder', "Controls sorting order of files and folders in the explorer.")
 		},
+		'explorer.sortOrderOption': {
```

Comment:
How is it possibly to understand the difference between `explorer.sortOrder` setting and `explorer.sortOrderOption`?

Reply:
Sort Order Option further refines the sort order. I was hoping the description made that clear - but can certainly reword if it doesn't.

![image](https://user-images.githubusercontent.com/873990/80404804-99701780-8876-11ea-8c9a-e7a222ea9e05.png)